linters.go

```diff
@@ -182,6 +182,25 @@ func defaultEnabled() []string {
 	return enabled
 }
 
+func validateLinters(linters map[string]*Linter, config *Config) error {
+	var unknownLinters []string
+	for name, _ := range linters {
+		if _, isDefault := defaultLinters[name]; !isDefault {
+			if _, isCustom := config.Linters[name]; !isCustom {
+				unknownLinters = append(unknownLinters, name)
+			}
+		}
+	}
+	switch len(unknownLinters) {
+	case 0:
+		return nil
+	case 1:
+		return fmt.Errorf("unknown linter: %s", unknownLinters[0])
+	default:
+		return fmt.Errorf("unknown linters: %s", strings.Join(unknownLinters, ", "))
```

Comment:
Just use this error rather than having a switch statement; it is much clearer to read the code.